### PR TITLE
WandB Integration

### PR DESCRIPTION
**Pull request by Bhuvanesh-Verma**
1. Word Analogy task function added to utils.
2. [WandB](https://wandb.ai/) framework for project visualization is added.
3. questions-words.txt dataset added. It will be used for word analogy evaluation task.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## run : pip install -r requirements.txt
+sklearn==0.0
+tqdm==4.61
+pyyaml==5.4
+torch==1.8
+wandb==0.11
```
Comment (TamaraAtanasoska) [inline requirements.txt:6]: thanks for adding the requirements.txt!

File: utils.py [hunk 2]
```diff
@@ -25,6 +28,55 @@ def preprocess(text: string) -> list:
...
+
+    for word in vocab.get_vocab():
+        if word in words:
+            continue
+        word_idx = vocab.lookup_token(word)
+        word_idx = torch.LongTensor([word_idx]).to(device)
+        wvec = embeddings(word_idx)
+        diff1 = vectors[1] - vectors[0]
+        diff2 = wvec - vectors[2]
+        similarity = cosine_similarity(diff1, diff2)
```
Comment (TamaraAtanasoska) [inline utils.py:71]: I like the sklearn implementation too, I will use it as well 👍

**Review by TamaraAtanasoska — CHANGES_REQUESTED** [review]
Thanks for this great work! The word_analogy maybe we can merge to master once it is ready to work with data, otherwise 👍 or LGTM as people would say 🥳

**Comment by Bhuvanesh-Verma** [comment]
I think we should merge this as this is related to wandb integration. I will create another branch concerning word_analogy task.

**Pull request merged**